docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,20 +1,19 @@
 == AeroGear Digger Installation
-The AeroGear Digger OpenShift project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
+The AeroGear Digger project allows you to setup a Jenkins CI/CD server for building mobile applications using OpenShift containers.
 
 https://github.com/aerogear/aerogear-digger-installer[aerogear-digger-installer^] - GitHub repo with http://docs.ansible.com/ansible/index.html[Ansible^] based automation tool used to install AeroGear Digger.
 
 NOTE: For more details about Ansible playbooks and variables, please refer to http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs^].
 
 NOTE: It is possible to deploy Digger behind HTTP/HTTPS proxy that do not require authentication. see <<using-http-proxy>>
 
-Use the tool to  create an OpenShift project and deploy the following components in it:
+Use the tool to create an OpenShift project and deploy the following components in it:
 
 * A Jenkins server, used to run the build tasks
-* A Docker-formatted image with all tools required for Android application building
-* A Nagios server for health monitoring of installed components 
-* Android SDK
+* Docker-formatted images with all tools required for Android application building.
+* A Nagios server for health monitoring of installed components
 
-If you want iOS build capability, installer can optionally configure macOS servers with required tools. See <<macos-setup>>. These are not deployed in OpenShift project, but are connected as a Jenkins slaves to Jenkins master present in OpenShift project.
+If you want iOS build capability, installer can configure macOS server(s) with required tools. See <<macos-setup>>. These are not deployed in OpenShift project, but are connected as a Jenkins slaves to Jenkins master present in OpenShift project.
```

Comment:
lets not remove `optionally` here

Reply:
thought "can" and "want" implies optionality